kong/db/dao/init.lua

```diff
@@ -1482,6 +1482,22 @@ local function get_cache_key_value(name, key, fields)
 end
 
 
+local normalize_string
+do
+  local sha256       = utils.sha256_hex
+  local MAX_KEY_SIZE = 256
```

Comment:
Should we use a higher limit instead? We can safely go way over 256 and this can reduce the number of hashing we need to do.

Reply:
Do we have any suggestion? `384` or `448`?

Reply:
As dndx metioned, we should use number as big as we can, so I think 448 is a proper number?

512 > 448 = 256 + 128 + 64

Reply:
We need to reserve enough length for the rest of the components of the cache key, like `arg2` ~ `arg5`, `ws_id`

Reply:
adjust `MAX_KEY_SIZE` to `448`